src/group/stabchain/moved_point_selector.rs

```diff
@@ -53,6 +55,21 @@ impl<P, T> MovedPointSelector<P, T> for FixedBaseSelector<T> {
             .expect("Base was shorter than expected")
     }
 }
+///Struct for a base point selector that takes the first point to be moved.
+#[derive(Default, Debug, Copy, Clone)]
+pub(crate) struct FmpSelector;
```

Comment:
Smp maybe? For smallest moved point?

Reply:
Or maybe lmp stands for last moved point. I always thought of it as Largest Moved Point